src/utils/sentry.js

```diff
@@ -0,0 +1,23 @@
+const Sentry = require('@sentry/node');
+
+const logger = require('@asymmetrik/node-fhir-server-core').loggers.get();
```

Comment:
This file is a.lmost identifical to ./middleware/sentry file

Probably want to remove one and use the other.